discord/ext/ipc/client.py

```diff
@@ -21,13 +21,15 @@
 
 class Client:
     """Handles webserver side requests to the bot process.
-
-    :param host: The IP or host of the IPC server, defaults to localhost
-    :type host: ``str``, optional
-    :param port: The port of the IPC server. If not supplied the port will be found automatically, defaults to None
-    :type port: ``int``, optional
-    :param secret_key: The secret key for your IPC server. Must match the server secret_key or requests will not go ahead, defaults to None
-    :type secret_key: ``Union[str, bytes]``, optional
+    
+    Parameters
+    ----------
+    host: str
+        The IP or host of the IPC server, defaults to localhost
+    port: int
+        The port of the IPC server. If not supplied the port will be found automatically, defaults to None
+    secret_key: ``Union[str, bytes]``, optional
```

Comment:
```suggestion
    secret_key: Union[str, bytes]
```

Reply:
This will complain. But I'll figure out something

Reply:
What gives?

Reply:
Can't remember honestly, I'm busy tonight but I'll @ you with it when I work on this

Reply:
Simply applied your suggestion